src/components/ColumnMenuButton/ColumnMenuButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { MouseEvent } from 'react'
+import styles from './ColumnMenuButton.module.css'
+
+interface ColumnMenuButtonProps {
+  onClick: (e: MouseEvent) => void
+}
+
+export default function ColumnMenuButton({ onClick }: ColumnMenuButtonProps) {
+  return (
+    <div
+      className={styles.columnMenuButton}
+      onClick={e => {
+        e.stopPropagation()
+        onClick(e)
+      }}
```

Comment:
```suggestion
      onClick={onClick}
```

Saves a function allocation. More importantly let's the parent decide whether to stopPropagation or not.